authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHivePlugin.java

```diff
@@ -60,6 +60,37 @@ public static synchronized RangerAuthorizationHivePlugin getInstance(Map<String,
     return instance;
   }
 
+  /** Validate different Ranger metadata object */
+  @Override
+  public void validateRangerMetadataObject(List<String> names, RangerMetadataObject.Type type)
+      throws IllegalArgumentException {
+    Preconditions.checkArgument(
+        names != null, "Cannot create a Ranger metadata object with null names");
+    Preconditions.checkArgument(
+        !names.isEmpty(), "Cannot create a Ranger metadata object with no names");
+    Preconditions.checkArgument(
+        names.size() <= 3,
+        "Cannot create a Ranger metadata object with the name length which is greater than 3");
+    Preconditions.checkArgument(
+        type != null, "Cannot create a Ranger metadata object with no type");
+
+    Preconditions.checkArgument(
+        names.size() != 1 || type == RangerMetadataObject.Type.SCHEMA,
+        "If the length of names is 1, it must be the SCHEMA type");
```

Comment:
According to the condition, If the type is `RangerMetadataObject.Type.SCHEM`,  the names with any length is OK

Reply:
This code is right.